src/test/java/org/prebid/server/bidder/adocean/AdoceanBidderTest.java

```diff
@@ -88,56 +89,19 @@ public void makeHttpRequestsShouldReturnErrorIfImpExtCouldNotBeParsed() {
         assertThat(result.getErrors().get(0).getMessage()).startsWith("Cannot deserialize instance");
     }
 
-    @Test
-    public void makeHttpRequestsShouldSetReturnErrorIfRequestExists() {
-        // given
-        final BidRequest bidRequest = BidRequest.builder()
-                .user(User.builder()
-                        .ext(ExtUser.builder()
-                                .consent("COwK6gaOwK6gaFmAAAENAPCAAAAAAAAAAAAAAAAAAAAA."
-                                        + "IFoEUQQgAIQwgIwQABAEAAAAOIAACAIAAAAQAIAgEAACEAAAAAgAQBAAAAAAAGBAAgAAAAAA"
-                                        + "AFAAECAAAgAAQARAEQAAAAAJAAIAAgAAAYQEAAAQmAgBC3ZAYzUw").build())
-                        .build())
-                .imp(asList(Imp.builder()
-                                .id("ao-test")
-                                .banner(Banner.builder().format(singletonList(Format.builder().w(300).h(250).build()))
-                                        .id("banner_id").build()).ext(mapper.valueToTree(ExtPrebid.of(null,
-                                ExtImpAdocean.of("myao.adocean.pl", "tmYF.DMl7ZBq.Nqt2Bq4FutQTJfTpxCOmtNPZoQUDcL.G7",
-                                        "adoceanmyaozpniqismex")))).build(),
-                        Imp.builder()
-                                .id("ao-test")
-                                .banner(Banner.builder().format(singletonList(Format.builder().w(300).h(250).build()))
-                                        .id("banner_id").build()).ext(mapper.valueToTree(ExtPrebid.of(null,
-                                ExtImpAdocean.of("myao.adocean.pl", "tmYF.DMl7ZBq.Nqt2Bq4FutQTJfTpxCOmtNPZoQUDcL.G7",
-                                        "adoceanmyaozpniqis")))).build()))
-                .test(1)
-
-                .build();
-
-        // when
-        final Result<List<HttpRequest<Void>>> result = adoceanBidder.makeHttpRequests(bidRequest);
-
-        // then
-        assertThat(result.getErrors()).hasSize(1);
```

Comment:
This test is still cover use case. Just rename it to 
makeHttpRequestsShouldSetReturnErrorIfRequestIsDuplicate

Reply:
You commented that we do not need to throw exception. In Go they do not add error in this case